test/browser_tests/cucumber/features/filterable-page.feature

```diff
@@ -0,0 +1,18 @@
+
+Feature: Filterable
+  As a user of cf.gov
+  I expect to see all results when I haven't selected a filter
+
+  Scenario: Sublanding filterable page
+  	Given I goto a sublanding filterable page
+  	When I do not select a filter on the sublanding filterable page
```

Comment:
yeah, we need to manage state within a scenario, I am working on this. I think think for now just make the step definition /I do not select a filter on the (.*) page/. We have the ability to use regexes in the step definitions.